starter-code/pizza.js

```diff
@@ -1 +1,34 @@
 // Write your Pizza Builder JavaScript in this file.
+
+
+$("button.btn-pepperonni").click(function(event) {
+  $('.pep').toggle();
+  $(this).toggleClass("active");
+  $(".price ul li:first-child").toggle();
+});
+$("button.btn-mushrooms").click(function(event) {
+  $('.mushroom').toggle();
+  $(this).toggleClass("active");
+  $(".price ul li:nth-child(2)").toggle();
+});
+
+$("button.btn-green-peppers").click(function(event) {
+  $('.green-pepper').toggle();
+  $(this).toggleClass("active");
+  $(".price ul li:nth-child(3)").toggle();
+});
+
+
+$("*").removeClass("crust-gluten-free");
```

Comment:
You don't have to select everything. Because you're removing the class `crust-gluten-free`, you can instead do `$('.crust-gluten-free').removeClass('crust-gluten-free')`.